packages/sdk/wallets/wallet-ledger/src/ledger-signer.ts

```diff
@@ -2,27 +2,27 @@ import { ensureLeading0x, trimLeading0x } from '@celo/base/lib/address'
 import { RLPEncodedTx, Signer } from '@celo/connect'
 import { EIP712TypedData, structHash } from '@celo/utils/lib/sign-typed-data-utils'
 import { TransportStatusError } from '@ledgerhq/errors'
+import Ledger from '@ledgerhq/hw-app-eth'
 import debugFactory from 'debug'
 import * as ethUtil from 'ethereumjs-util'
 import { transportErrorFriendlyMessage } from './ledger-utils'
 import { AddressValidation } from './ledger-wallet'
 import { compareLedgerAppVersions, tokenInfoByAddressAndChainId } from './tokens'
-
 const debug = debugFactory('kit:wallet:ledger')
```

Comment:
Nit: suggest putting back the deleted empty line.
```suggestion

const debug = debugFactory('kit:wallet:ledger')
```